tests/sentry/deletions/test_alert_rule.py

```diff
@@ -68,7 +70,9 @@ def test_simple(self):
         assert not Incident.objects.filter(id=incident.id).exists()
         assert not IncidentProject.objects.filter(incident=incident, project=self.project).exists()
         assert not IncidentGroupOpenPeriod.objects.filter(
-            incident_id=incident.id, group_open_period=group_open_period
+            incident_id=incident.id,
+            incident_identifier=incident.identifier,
```

Comment:
Is the idea that we need this after the `Incident` table is gone?

Reply:
yeah exactly, the `IncidentGroupOpenPeriod` table exists so that we don't have to rely on a "legacy" table